docs/api/app_commands.rst

```diff
@@ -0,0 +1,205 @@
+.. SPDX-License-Identifier: MIT
+
+.. currentmodule:: disnake
+
+Application Commands
+====================
+
+This section documents everything related to Discord's application commands.
```

Comment:
a link here to https://discord.com/developers/docs/interactions/application-commands would be nice

Reply:
Done in [`d15ddd7`](https://github.com/DisnakeDev/disnake/pull/722/commits/d15ddd70e20c611070a3445379763a80b07a8c13)